src/win32/base/tsf_registrar.cc

```diff
@@ -73,7 +73,7 @@ const GUID kCategories[] = {
 //  5. Click "Details" in the "Text services and input languages" frame, and;
 //  6. All installed prossors are enumerated in the "Installed services"
 //     frame.
-HRESULT TsfRegistrar::RegisterProfiles(const wchar_t *path, DWORD path_length) {
+HRESULT TsfRegistrar::RegisterProfiles(std::wstring_view resource_dll_path) {
```

Comment:
Please include the header file for `std::wstring_view`.

Reply:
Done.